openverse_catalog/dags/providers/provider_api_scripts/jamendo.py

```diff
@@ -184,27 +184,42 @@ def _get_foreign_identifier(media_data):
         return None
 
 
-def _get_audio_info(media_data):
-    """Parses audio URL, audio download URL, audio duration
-    If the audio does not allow download, we save the 'streaming'
-    URL as the `audio_url`
+def _remove_param_from_url(url: str, param: str) -> str:
+    """
+    Remove a parameter from a provided URL.
+    """
```

Comment:
This function looks potentially useful for many providers!

Reply:
That's a good point! If any of us comes across another scenario where this is needed, we should make it a general utility function 🙂